t/BikeSharingCitiBikeNYC.t

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::BikeSharing::CitiBikeNYC )],
+    'citi bike ny' => test_spice(
+        '/js/spice/bike_sharing/citi_bike_nyc/ny',
+        call_type => 'include',
+        caller => 'DDG::Spice::BikeSharing::CitiBikeNYC',
+    is_cached => 1,
+    ),
```

Comment:
@iambibhas thanks for the idea... I've just commited a change that takes a reference point in one of the specified places, and calculates the distance to it when sorting the stations